packages/core/src/App/Containers/RealAccountSignup/real-account-signup.jsx

```diff
@@ -18,9 +17,12 @@ import SignupErrorContent from './signup-error-content.jsx';
 import StatusDialogContainer from './status-dialog-container.jsx';
 import { Analytics } from '@deriv-com/analytics';
 
-import 'Sass/account-wizard.scss';
```

Comment:
This is going to break the styling of the real account signup and manage accounts. Would you help to look into it, you might need to get some help from someone from Accounts team.

cc: @amina-deriv 

<img width="1013" alt="Screenshot 2024-02-12 at 10 48 06 AM" src="https://github.com/binary-com/deriv-app/assets/70938039/7646f087-bbe2-4158-881a-94cda9ff8aee">

Reply:
@ahmadtaimoor-deriv @matin-deriv This is effecting trading assesment modal styles as well
<img width="1416" alt="Screenshot 2024-02-12 at 12 54 24 PM" src="https://github.com/binary-com/deriv-app/assets/84661147/2f52f634-ab9c-4d57-aa48-64763d1ac56a">

Reply:
It is reverted back to how it was in read-account-signup and scss is added inside the modal. Thanks